SmileAuth/Classes/SmileAuthenticator.m

```diff
@@ -304,4 +304,7 @@ +(void)clearPassword{
     [[SmileAuthenticator sharedInstance].keychainWrapper writeToKeychain];
 }
 
+-(void)changeAuthentication:(BOOL)newAuth {
+    _isAuthenticated = NO;
+}
```

Comment:
Sorry, that should have been set to the parameter, but in my project I was just using False. I'll change it